src/axolotl/train.py

```diff
@@ -80,10 +80,6 @@ def train(
 
     model.config.use_cache = False
 
-    if torch.__version__ >= "2" and sys.platform != "win32":
```

Comment:
do we still need to add this check on validate config?

Reply:
No I think we can let HF handle it